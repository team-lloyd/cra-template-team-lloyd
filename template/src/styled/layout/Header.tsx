import styled from 'styled-components';

export const HeaderLayout = styled.header`
  height: 64px;
  width: 100%;
  flex-direction: row;
  align-items: center;
  padding-left: 16px;
  padding-right: 16px;
`;
