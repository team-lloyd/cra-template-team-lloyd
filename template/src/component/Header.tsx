import React from 'react';
import { HeaderLayout } from '../styled/layout/Header';
import { H6 } from '../styled/typo/Heading';
import { title } from '../config/app.json';

const Header: React.FC = function Header() {
  return (
    <HeaderLayout>
      <H6>{title}</H6>
    </HeaderLayout>
  );
};

export default Header;
