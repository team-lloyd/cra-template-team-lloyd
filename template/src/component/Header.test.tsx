import React from 'react';
import { shallow } from 'enzyme';
import Header from './Header';

it('renders without crashing', () => {
  const snapshot = shallow(<Header />);
  expect(snapshot).toMatchSnapshot();
});
