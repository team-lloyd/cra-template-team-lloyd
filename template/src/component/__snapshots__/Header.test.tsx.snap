// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<styled.header>
  <styled.h6>
    TEAM LLOYD Template App
  </styled.h6>
</styled.header>
`;
